charts/memgraph/templates/statefulset.yaml

```diff
@@ -27,57 +27,120 @@ spec:
         {{- toYaml . | nindent 4 }}
       {{- end }}
     spec:
+      terminationGracePeriodSeconds: 1800
```

Comment:
Can we move this value also in the values file? 

BTW, did you have issues with the default termination grace period?

Reply:
Shutdown of Memgraph on Big Datasets of 1TB+ can take up to 20 min. If terminationGracePeriodSeconds is left at default (30 sec) then the pod will terminate and shutdown will not happen gracefully.